### PR TITLE
Qualys Scan csv format support

### PR DESCRIPTION
**Pull request by 37b**
Added CSV format support for the Qualys Parser.

### CODE REVIEW

File: docs/content/en/integrations/import.md [hunk 1]
```diff
@@ -385,16 +385,15 @@ XML file from IBM App Scanner.
 
 XML Scan Result File from Immuniweb Scan.
 
-IntSights Report
---------------
+###IntSights Report
```
Comment (damiencarol) [inline docs/content/en/integrations/import.md:388]:
```suggestion
### IntSights Report
```

File: dojo/fixtures/initial_surveys.json [hunk 2]
```diff
@@ -486,7 +486,7 @@
       "modified": "2015-03-30T20:11:30Z",
       "optional": false,
       "order": 1,
-      "polymorphic_ctype": 18
+      "polymorphic_ctype": 77
```
Comment (damiencarol) [inline dojo/fixtures/initial_surveys.json:489]: I don't see the link between this PR and this file

**Comment by damiencarol** [comment]
@37b I see modifications on another parser than Qualys Scan. Are you missing a rebase here?

**Comment by 37b** [comment]
@damiencarol Very weird..will resubmit with a clean start

**Pull request closed without merging**